recipe/meta.yaml

```diff
@@ -19,19 +19,29 @@ requirements:
     - future
     - python
     - pip
-    - numpy
-    - scipy
+    - numpy ==1.11.3 py27hab51878_1207  # [win and py2k]
+    - numpy  # [not (win and py2k)]
+    - scipy ==1.2.1 py27h4c3ab11_0  # [win and py2k]
+    - scipy  # [not (win and py2k)]
+    - libblas ==3.8.0 8_mkl  # [win and py2k]
+    - libcblas ==3.8.0 8_mkl  # [win and py2k]
+    - liblapack ==3.8.0 8_mkl  # [win and py2k]
   run:
+    - future
     - python
-    - numpy
-    - scipy
+    - numpy ==1.11.3 py27hab51878_1207  # [win and py2k]
+    - numpy  # [not (win and py2k)]
+    - scipy ==1.2.1 py27h4c3ab11_0  # [win and py2k]
+    - scipy  # [not (win and py2k)]
     - gmsh <4.0   # [not (win and (py2k or x86))]
+    - mpich       # [not win]
+    - mpi4py      # [not win]
+    - petsc4py    # [not win]
```

Comment:
Of all the changes, this is the only line you seem to need. I would try to revert all the pinnings and see what happens.